addons/mail/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'Discuss',
-    'version': '1.8',
+    'version': '1.9',
```

Comment:
Note that mail was already bumped in 15.3 so no need to bump it again. Indeed we currently don't really handle "between saas" versions so in the end having a single file for a given saas is sufficient.

Reply:
Good to know. I've just changed back to 1.8 and I have merged the upgrade into the existing one.